venv/lib/python3.9/site-packages/pip/_internal/commands/freeze.py

```
@@ -0,0 +1,104 @@
+            help='Do not skip these packages in the output:'
+                 ' {}'.format(', '.join(DEV_PKGS)))
+        self.cmd_opts.add_option(
+            '--exclude-editable',
+            dest='exclude_editable',
+            action='store_true',
+            help='Exclude editable package from output.')
+        self.cmd_opts.add_option(cmdoptions.list_exclude())
+
+        self.parser.insert_option_group(0, self.cmd_opts)
+
+    def run(self, options, args):
```

Ruff (0.14.8)

Unused method argument: `args`

(ARG002)